### PR TITLE
add publicPath, filenameHashing, productionSourceMap, css, and devServer options

### LINKED ISSUE #50: Public Path option
Would it be possible to add the public path option from vue directly to this plugin? I do not think there is another way to set this. When using the `vue.config.js` it will disable all the other options set in `workspace.json`.

### PR DESCRIPTION
**Pull request by BuckyMaler**
Having these options available in `workspace.json` is equivalent to Vue CLI's support for these options in `package.json`. I tested each option running `nx serve` and `nx build`. Below is a screenshot of a Vue app proxying an Express app. You can see the response from the `/api` route in the browser console.

Screen Shot 2020-07-14 at 10 58 50 PM

Vue CLI loads these options [here](https://github.com/vuejs/vue-cli/blob/v4.3.1/packages/%40vue/cli-service/lib/Service.js#L316). The default values can be seen [here](https://github.com/vuejs/vue-cli/blob/v4.3.1/packages/%40vue/cli-service/lib/options.js#L81) or in the [docs](https://cli.vuejs.org/config/#vue-config-js).

Closes #50

### CODE REVIEW

File: libs/vue/README.md [hunk 1]
```diff
@@ -82,36 +82,49 @@ nx build my-app
...
+| `--filenameHashing`          | `false`       | Generated static assets contains hashes in their filenames for better caching control.                                                                                                                                                |
+| `--productionSourceMap`      | `false`       | Setting this to `false` can speed up production builds if you don't need source maps for production.                                                                                                                                  |
+| `css.requireModuleExtension` | `true`        | By default, only files that ends in `*.module.[ext]` are treated as CSS modules. Setting this to `false` will allow you to drop `.module` in the filenames and treat all `*.(css\|scss\|sass\|less\|styl(us)?)` files as CSS modules. |
+| `css.extract`                | `true`        | Whether to extract CSS in your components into a standalone CSS files (instead of inlined in JavaScript and injected dynamically).                                                                                                    |
```
Comment (ZachJW34) [inline libs/vue/README.md:125]: `css.extract` is false by default in both schemas

File: libs/vue/README.md [hunk 2]
```diff
@@ -82,36 +82,49 @@ nx build my-app
...
+| `--main`                     | -             | The full path for the main entry point to the app, relative to the current workspace.                                                                                                                                                 |
+| `--tsConfig`                 | -             | The full path for the TypeScript configuration file, relative to the current workspace.                                                                                                                                               |
+| `--publicPath`               | `/`           | The base URL your application bundle will be deployed at.                                                                                                                                                                             |
+| `--filenameHashing`          | `false`       | Generated static assets contains hashes in their filenames for better caching control.                                                                                                                                                |
```
Comment (ZachJW34) [inline libs/vue/README.md:122]: contains -> contain

File: libs/vue/README.md [hunk 3]
```diff
@@ -82,36 +82,49 @@ nx build my-app
...
+| `--productionSourceMap`      | `false`       | Setting this to `false` can speed up production builds if you don't need source maps for production.                                                                                                                                  |
+| `css.requireModuleExtension` | `true`        | By default, only files that ends in `*.module.[ext]` are treated as CSS modules. Setting this to `false` will allow you to drop `.module` in the filenames and treat all `*.(css\|scss\|sass\|less\|styl(us)?)` files as CSS modules. |
```
Comment (ZachJW34) [inline libs/vue/README.md:124]: ends -> end

File: libs/vue/README.md (hunk 1 above)
Comment (ZachJW34) [inline libs/vue/README.md:125]: standalone CSS files -> standalone CSS file

File: libs/vue/src/builders/dev-server/schema.d.ts [hunk 4]
```diff
@@ -12,4 +12,12 @@ export interface DevServerBuilderSchema extends JsonObject {
   skipPlugins?: string;
   browserTarget: string;
   watch: boolean;
+  publicPath?: string;
+  css: {
+    requireModuleExtension?: boolean;
+    extract?: boolean | object;
+    sourceMap?: boolean;
+    loaderOptions: object;
```
Comment (ZachJW34) [inline libs/vue/src/builders/dev-server/schema.d.ts:20]: Should this property be optional?

Comment (BuckyMaler) [reply libs/vue/src/builders/dev-server/schema.d.ts:20]: No. Even without a default value specified in the schema it defaults to an empty object.

**Pull request merged**